docs/language-reference/inference.md

```diff
@@ -1,126 +1,147 @@
 # Inference methods
 
-TreePPL supports a continuously growing number of inference methods.
+TreePPL supports number of inference methods.
```

Comment:
There's a word missing here, "[...] supports **a** number of inference [...]" or "[...] supports **several** inference [...]"